### PR TITLE
Fixed AQL issue in Ethercis by making manifestation pull back flatten…

### PR DESCRIPTION
**Pull request by freshehr**
fix for #12 -changed AQL to use FLAT DV_TEXT format

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,13 +1,82 @@
...
   <component name="MavenProjectsManager">
     <option name="originalFiles">
       <list>
         <option value="$PROJECT_DIR$/pom.xml" />
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/out" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_6" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (lapauk) [inline .idea/misc.xml]: JDK_1_6  ???

Comment (freshehr) [reply .idea/misc.xml]: Arrgh - no idea!! I had some setup issues when I renamed the repo- must have screwed up on the git ignore.

**Comment by freshehr** [comment]
@lapauk   I think I have removed the .idea files now.

**Comment by lapauk** [comment]
I've put the changes into PR #21 will close this...

**Pull request closed without merging**